tools/bs_parser_hevc/src/hevc2_parser.cpp

```diff
@@ -759,7 +759,10 @@ Bs32u Parser::ParseSSDSubmit(SDThread*& pSDT, NALU* AU)
 
         sdpar.Emulation = true;
 
-        TargetRBSP = Bs32u(CurRBSP / m_sdt.size());
+        if (m_sdt.empty())
```

Comment:
no need of this, I synced with Evgeniy, size of vector can't be zero here

Reply:
This check was removed